docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -400,27 +418,24 @@ ALPN enables more efficient communication by allowing the client to indicate its
 
```

Comment:
The second sentence in the previous paragraph mostly just repeats the first sentence, so I think a large part of it can be cut out. Also, "efficient communication" is pretty vague. What sort of communication are we talking about? And between what SW/HW components?